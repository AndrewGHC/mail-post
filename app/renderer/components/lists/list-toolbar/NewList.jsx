// @flow
import React from 'react';
import Button from 'material-ui/Button';
import AddIcon from 'material-ui-icons/Add';

import { csvImport } from '../../../ipc/send';

const NewList = () => (
  <div>
    <Button
      onClick={() => csvImport()}
      color="primary"
      variant="raised"
      data-test="import-csv"
    >
      <AddIcon />
      New List
    </Button>
  </div>
);

export default NewList;
