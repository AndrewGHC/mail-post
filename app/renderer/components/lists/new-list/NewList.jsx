// @flow
import React from 'react';
import { csvImport } from '../../../utils';

const NewList = () => (
  <div>
    <button onClick={() => csvImport()} data-test="import-csv">
      Import CSV
    </button>
  </div>
);

export default NewList;
