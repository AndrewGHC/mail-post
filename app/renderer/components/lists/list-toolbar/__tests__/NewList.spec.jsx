import React from 'react';
import { shallow } from 'enzyme';

import NewList from '../NewList';

describe('NewList', () => {
  it('renders elements', async () => {
    const wrapper = shallow(<NewList />);
    expect(wrapper).toMatchSnapshot();
  });
});
