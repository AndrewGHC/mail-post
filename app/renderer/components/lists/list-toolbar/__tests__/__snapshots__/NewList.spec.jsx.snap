// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewList renders elements 1`] = `
<div>
  <button
    data-test="import-csv"
    onClick={[Function]}
  >
    Import CSV
  </button>
</div>
`;
